src/org/spdx/crossref/UrlConstants.java

```diff
@@ -32,4 +32,6 @@
 	public static final Integer CROSS_REF_INDEX_ISWAYBACKLINK = 3;
 	public static final Integer CROSS_REF_INDEX_MATCH = 4;
 	public static final Integer CROSS_REF_INDEX_TIMESTAMP = 5;
+	public static final Integer CROSS_REF_LICENSE_TEXT_START_CHAR_COUNT = 80;
+	public static final Integer CROSS_REF_LICENSE_TEXT_END_CHAR_COUNT = 60;
```

Comment:
Is the `CROSS_REF_LICENSE_TEXT_START_CHAR_COUNT` and `CROSS_REF_LICENSE_TEXT_END_CHAR_COUNT` still in use?  If not, these should be removed.

Reply:
I have removed them.